plugins/modules/win_firewall.py

```diff
@@ -24,6 +24,18 @@
     - Set state of firewall for given profile.
     type: str
     choices: [ disabled, enabled ]
+  inbound_action:
+    description:
+    - Allow or block inbound network traffic in the profile.
+    - NotConfigured is valid when configuring a GPO.
```

Comment:
Same here should match the actual choice and enclosed with `C()`, e.g. `C(not_configured) is valid when configuring a GPO.`